src/pybind/mgr/dashboard/README.rst

```diff
@@ -69,6 +69,41 @@ The password will be stored as a hash using ``bcrypt``.
 
 The Dashboard's WebUI should then be reachable on TCP port 8080.
 
+Enabling the Rados Gateway management frontend
+----------------------------------------------
+
+If you want to use the Rados Gateway  management functionality of the
+dashboard, you will need to provide credentials.  If you do not have a user
```

Comment:
For consistency with the rest of this document, please add just one whitespace after a full-stop.

Reply:
I also wonder if it would make sense to emphasize that this API currently only supports communicating with a single RGW instance and that in a configuration with severals zones, it'll use the IP/port of the master node.